frontend/app/module/workstations/WorkstationsHomePage.tsx

```diff
@@ -2,21 +2,22 @@ import { Link } from "react-router-dom";
 
 import { NavBar } from "app/component/navbar/NavBar";
 
+import { t } from "@kiesraad/i18n";
 import { PageTitle } from "@kiesraad/ui";
 
 export function WorkstationsHomePage() {
   return (
     <>
       <PageTitle title="Invoerstations - Abacus" />
       <NavBar>
-        <Link to={"/elections#administrator"}>Verkiezingen</Link>
-        <Link to={"/users#administratorcoordinator"}>Gebruikers</Link>
-        <span className="active">Invoerstations</span>
-        <Link to={"/logs#administratorcoordinator"}>Logs</Link>
+        <Link to={"/elections#administrator"}>{t("election.name_plural")}</Link>
+        <Link to={"/users#administratorcoordinator"}>{t("users")}</Link>
+        <span className="active">{t("polling_stations")}</span>
```

Comment:
```suggestion
        <span className="active">{t("data_entry_stations")}</span>
```

Reply:
this is actually a nice example for not using translations in test files, would've caught the mismatch earlier